Source/Core/Core/Analytics.cpp

```diff
@@ -55,16 +57,21 @@ void DolphinAnalytics::ReloadConfig()
 {
   std::lock_guard<std::mutex> lk(m_reporter_mutex);
 
+  auto base_layer = Config::GetLayer(Config::LayerType::Base);
+  auto analytics_section = base_layer->GetOrCreateSection(Config::System::Main, "Analytics");
+  bool enabled = false;
+  analytics_section->Get("Enabled", &enabled, false);
```

Comment:
Again, can't help but notice how boilerplatey this is. 4 lines to replace something that took one before.